release/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "agent-release",
+  "version": "1.0.0",
+  "description": "Flow for releasing agent",
+  "main": "mkrelease.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "dependencies": {
+    "child_process": "^1.0.2",
```

Comment:
Nit: Isn't child_process part of built in node? I don't _think_ we need to explicitly add it here